### DISCUSSION TITLE
Fuzzy Skin on selected surface or on outter layer only

### DISCUSSION THREAD

**Discussion by ReggieLucas** [OrcaSlicer/OrcaSlicer#5544] [Ideas]
When you're using fuzzy skin, it applies it to all outside walls. If we use a modifier, it adds fuzzy skin to the interior of the walls, and that cause structural issues.
It'll be nice if some how, we can select the faces that the fuzzy skin will be applied.
Thank you for all your hard work and dedication to make this app a big success.

**Comment by dantetemp** [comment]
that would be awesome!

**Comment by UrbanAstronomy** [comment]
Something similar to multimaterial painting but applied to fuzzy skin would be ideal.  I have several models that would benefit from having a texture on exterior surfaces but following contour often wraps around to interior walls at different layers.  This would be particularly beneficial where you have a pocket that you're putting a heatset insert into.  I don't want that surface to have any texture but it's still technically outside contour.

**Comment by RF47** [comment]
check this out:
https://github.com/SoftFever/OrcaSlicer/pull/9979
![image](https://github.com/user-attachments/assets/2bbd08f0-6b0d-4ae6-b65c-f22507708b67)
you can download the executable from here:
https://github.com/SoftFever/OrcaSlicer/actions/runs/16066091477?pr=9979
